climt/_core/constants.py

```diff
@@ -0,0 +1,303 @@
+from sympl import default_constants, DataArray
+from copy import deepcopy
+
+constant_name_map = {
+
+    'gas_constant_of_vapor_phase': 'gas_constant_of_water_vapor',
+    'heat_capacity_of_vapor_phase': 'heat_capacity_of_water_vapor_at_constant_pressure',
+    'specific_enthalpy_of_vapor_phase': 'specific_enthalpy_of_water_vapor',
+    'stellar_irradiance': 'solar_constant',
+    'density_of_liquid_phase': 'density_of_liquid_water',
+    'heat_capacity_of_liquid_phase': 'heat_capacity_of_liquid_water',
+    'latent_heat_of_condensation': 'latent_heat_of_vaporization_of_water',
+    'freezing_temperature_of_liquid_phase': 'freezing_temperature_of_liquid_water',
+    'density_of_solid_phase_as_ice': 'density_of_ice',
+    'density_of_solid_phase_as_snow': 'density_of_snow',
+    'heat_capacity_of_solid_phase_as_ice': 'heat_capacity_of_ice',
+    'heat_capacity_of_solid_phase_as_snow': 'heat_capacity_of_snow',
+    'latent_heat_of_fusion': 'latent_heat_of_fusion_of_water',
+    'freezing_temperature_of_liquid_phase': 'freezing_temperature_of_liquid_water',
+    'thermal_conductivity_of_solid_phase_as_ice': 'thermal_conductivity_of_ice',
+    'thermal_conductivity_of_solid_phase_as_snow': 'thermal_conductivity_of_snow',
+    'thermal_conductivity_of_liquid_phase': 'thermal_conductivity_of_liquid_water'
+}
+
+default_categories = dict(
+    planetary_constants=set(('gravitational_acceleration',
```

Comment:
Instead of set(), should probably use sorted(list()). You don't need the performance of set here, and a sorted list makes sure the print-out is alphabetical.

Reply:
I need it because I'm using the union operation in the code. However, I have added sorted(list())
in the list_available_constants() function.